plugins/modules/idrac_server_config_profile.py

```diff
@@ -866,6 +1007,71 @@ def validate_scp_components(module, idrac):
                 module.exit_json(msg=msg, failed=True)
 
 
+def is_check_idrac_latest(firmware_version):
+    if LooseVersion(firmware_version) >= MINIMUM_SUPPORTED_FIRMWARE_VERSION:
+        return True
+    return False
+
+
+def import_custom_defaults(module, idrac):
+    import_buffer = module.params.get("import_buffer")
+    command = module.params["command"]
+    cds_diff = compare_custom_default_configs(module, idrac)
+    check_mode_custom_defaults(module, cds_diff)
+    if not cds_diff:
+        module.exit_json(msg=NO_CHANGES_FOUND, changed=False)
+    share = {}
+    if not import_buffer:
```

Comment:
can we try to avoid statements in a negative sense? makes it hard to understand the logic. Apply to all the place where we have written like `if ABC is not None` or `not ABC`.

Reply:
Updated